src/components/Settings.tsx

```diff
@@ -138,14 +167,15 @@ export default function Settings() {
      <p className="text-sm mb-6 text-green-600 font-bold">{showUserMessage && usermessage}</p>
       <label className="label sr-only" htmlFor="username">Name</label>
         <input {...register('username', { required: true, minLength: 3, maxLength: 20 })} id="username" name="username" defaultValue={user.username} className="input appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-t-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm" type="text" />
```

Comment:
I know two people called Bo. Maybe reduce the required length to 2? And why cap at 20?

Reply:
No reason, just seems unlikely. I can reduce the minLength requirement.